src/vespa/vector-app/services.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<services version="1.0" xmlns:deploy="vespa" xmlns:preprocess="properties">
+    <container id="default" version="1.0">
+        <document-api/>
+        <search/>
+        <nodes>
+            <node hostalias="node1" />
+        </nodes>
+    </container>
+
+    <content id="vector" version="1.0">
+        <redundancy>2</redundancy>
+        <documents>
+            <document type="vector" mode="index" />
+        </documents>
+        <nodes>
+            <node hostalias="node1" distribution-key="0" />
+        </nodes>
+    </content>
+
```

Comment:
Remove empty lines.

Also I think redundancy can be 1 and be faster, but I could be wrong ;)

Reply:
Removed empty lines (except the last one)